docs/topics/wildfly-swarm/docs/reference/fractions/undertow.adoc

```diff
@@ -757,9 +829,15 @@ The number of requests this listener has served
 swarm.undertow.servers._KEY_.https-listeners._KEY_.request-parse-timeout:: 
 The maximum amount of time (in milliseconds) that can be spent parsing the request
 
+swarm.undertow.servers._KEY_.https-listeners._KEY_.require-host-http11:: 
+Require that all HTTP/1.1 requests have a 'Host' header, as per the RFC. IF the request does not include this header it will be rejected with a 403.
```

Comment:
singled quoted values and HTTP response codes should use backticks

Reply:
Note to self: not easily fixable in our sources.